src/com/massivecraft/factions/cmd/CmdFactionsPowerBoost.java

```diff
@@ -42,46 +46,98 @@ public CmdFactionsPowerBoost()
 	// -------------------------------------------- //
 	// OVERRIDE
 	// -------------------------------------------- //
-	
-	@Override
-	public void perform() throws MassiveException
+
+	@Override public void perform() throws MassiveException
```

Comment:
Blame IntelliJ. I had no control over how it formatted everything from their style template D;